### ISSUE TITLE
adding new authors form & process

### ISSUE THREAD

**Issue by matchaginseng** [writingatlas/feedback#4]
When adding a new story, the list of authors isn't updated immediately because it has to be approved. It would be helpful to have a screen that tells the user this (as well as the approximate amount of time it would take to get approved, and also send an email to the user once their author addition is approved). 

Would also be more convenient if the dropdown allowed the user to type in some values, and filter the names by those values, instead of being a pure select dropdown. 

Also, on Chrome, when you click on the link to add a new author in the story submitting form, and you select the option to open a new tab, the form looks like this (in short, gets rid of the formatting): 
<img width="916" alt="image" src="https://github.com/writingatlas/feedback/assets/55040116/94eb1560-e5b4-4ada-a05c-e0940322495c">
